R/tmb-methods.R

```diff
@@ -13,7 +13,7 @@
 #'
 #' @examples
 #' formula <- FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID)
-#' object <- h_mmrm_tmb(formula, fev_data, weights = rep(1, nrow(fev_data)))
+#' object <- fit_mmrm(formula, fev_data, weights = rep(1, nrow(fev_data)))
```

Comment:
unrelated but same file therefore linter checks were triggered:
guess needs a `# nolint` at end of line 105

Reply:
ah, now I get it - we cant even comment on line 105 directly i.e. linting issues. x/

Reply:
Bit of a riddle to me. There is a random `# nolint` is in line 106. It is also inconsistent with other S3 method definitions (they are not cought by the lintr). See eg line 89.

Reply:
seems to be solved now